doc/code_of_conduct_enforcement_manual.rst

```diff
@@ -0,0 +1,60 @@
+.. _Zephyr_Code_of_Conduct_Enforcement_Manual
```

Comment:
All documents need to be included in the site's table of contents or the doc build will complain about orphan docs.